packages/perseus/src/util/graphie.ts

```diff
@@ -1346,7 +1346,7 @@ export class Graphie {
         return this.drawingTransform().unscaleVector(point);
     };
 
-    private processAttributes(attrs: any) {
+    private processAttributes(attrs: Record<string, any> | undefined) {
```

Comment:
**question**: Should this type be the `StyleAttributes` type?

Reply:
I'm not confident in giving it a type yet. The callers of `processAttributes` are passing `any` to it, so it's hard to tell what the actual type should be. I'll come back to this in a future PR.